test/e2e/run_userns_test.go

```diff
@@ -42,6 +42,31 @@ var _ = Describe("Podman UserNS support", func() {
 
 	})
 
+	// Note: Lot of tests for build with --userns=auto are already there in buildah
+	// but they are skipped in podman CI because bud tests are executed in rootfull
+	// environment ( where mappings for the `containers` user is not present in /etc/subuid )
+	// causing them to skip hence this is a redundant test for sanity to make sure
+	// we don't break this feature for podman-remote.
+	It("podman build with --userns=auto", func() {
+		u, err := user.Current()
+		Expect(err).To(BeNil())
+		name := u.Name
+		if name == "root" {
+			name = "containers"
+		}
+		content, err := ioutil.ReadFile("/etc/subuid")
+		if err != nil {
+			Skip("cannot read /etc/subuid")
+		}
+		if !strings.Contains(string(content), name) {
+			Skip("cannot find mappings for the current user")
+		}
+		session := podmanTest.Podman([]string{"build", "-f", "build/Containerfile.userns-auto", "-t", "test", "--userns=auto"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		Expect(session.OutputToString()).To(ContainSubstring("1024"))
```

Comment:
Can you elaborate on the magic string 1024?

Reply:
@vrothberg This should be the size or length of the range of user IDs that is mapped between the two user namespaces, this defaults to 1024. Similar check is also in other `--userns=auto` tests.

Reply:
it is `AutoUserNsMinSize` in c/storage/store.go

Reply:
Fix this and we can merge.

Reply:
Added a comment. that explains this constant.

Reply:
Why not use `storage.AutoUserNsMinSize` directly?

Reply:
> Why not use `storage.AutoUserNsMinSize` directly?

@vrothberg SGTM